redirects/pages.yml

```diff
@@ -668,4 +668,13 @@
 - from:
     - /docs/kmm-configure-sqldelight-for-data-storage.html
     - /docs/multiplatform-mobile-configure-sqldelight-for-data-storage.html
-  to: https://play.kotlinlang.org/hands-on/Networking%20and%20Data%20Storage%20with%20Kotlin%20Multiplatfrom%20Mobile/
\ No newline at end of file
+  to: https://play.kotlinlang.org/hands-on/Networking%20and%20Data%20Storage%20with%20Kotlin%20Multiplatfrom%20Mobile/
+
+- from: /docs/kotlin-nights-branding.md
```

Comment:
I think, that WH will be enough, or it's better to writer them here explicitly?

Reply:
You are right, i'll fix it.